DataFormats/MuonReco/interface/EmulatedME0Segment.h

```diff
@@ -0,0 +1,76 @@
+#ifndef GEMRecHit_EmulatedME0Segment_h
```

Comment:
the include guards are suggested to include the subsystem and package in the name:
``` DataFormats_MuonReco_EmulatedME0Segment_h ```